### PR TITLE
[FIX] PHPDoc & clean code with scrutinizer

### PR DESCRIPTION
**Pull request by Th3Mouk**

### CODE REVIEW

File: Command/RenderBlockCommand.php [hunk 1]
```diff
@@ -53,8 +53,6 @@ public function execute(InputInterface $input, OutputInterface $output)
             throw new \RuntimeException('The service does not implement the CmsManagerInterface');
         }
 
-        $page = $manager->getPageById($input->getArgument('page_id'));
```
Comment (Soullivaneuh) [inline Command/RenderBlockCommand.php]: Why removing this?

Comment (rande) [inline Command/RenderBlockCommand.php]: This code is required as it load some data

Comment (Th3Mouk) [inline Command/RenderBlockCommand.php]: IMO this variable is unused, i missed some reasoning ?

Comment (rande) [inline Command/RenderBlockCommand.php]: The variablie is not used, but the getPageById load internal data.

**Comment by Th3Mouk** [comment]
@rande there is an interest in correcting the phpdoc for you or not?

**Comment by rande** [comment]
Yes, indeed ;)

File: Command/RenderBlockCommand.php (hunk 1 above)
Comment (Th3Mouk) [inline Command/RenderBlockCommand.php]: sorry i feel like a monkey at the end of afternoon =)

File: Model/PageInterface.php [hunk 2]
```diff
@@ -244,7 +244,7 @@ public function addChildren(PageInterface $children);
     /**
      * Get children.
      *
-     * @return array
+     * @return Doctrine\Common\Collections\ArrayCollection|array
```
Comment (OskarStark) [inline Model/PageInterface.php]: please use a ``use`` statement here

File: Model/PageInterface.php [hunk 3]
```diff
@@ -258,7 +258,7 @@ public function addBlocks(PageBlockInterface $block);
     /**
      * Get blocs.
      *
-     * @return array
+     * @return Doctrine\Common\Collections\ArrayCollection|array
```
Comment (OskarStark) [inline Model/PageInterface.php]: same here

File: Command/RenderBlockCommand.php (hunk 1 above)
Comment (OskarStark) [inline Command/RenderBlockCommand.php]: please revert this code change

**Comment by OskarStark** [comment]
Thank you for your contribution @Th3Mouk 

After applying the comments and some final review, we will merge your PR :+1: 

Oskar

**Comment by OskarStark** [comment]
Restarted travis

**Comment by Th3Mouk** [comment]
Should I do something ?

**Comment by OskarStark** [comment]
Thank you @Th3Mouk

**Pull request merged**